airflow/security/permissions.py

```diff
@@ -46,8 +46,12 @@
 RESOURCE_USER_LDAP_MODELVIEW = "UserLDAPModelView"
 RESOURCE_USER_OAUTH_MODELVIEW = "UserOAuthModelView"
 RESOURCE_USER_REMOTEUSER_MODELVIEW = "UserRemoteUserModelView"
+RESOURCE_ROLE_MODEL_VIEW = "RoleModelView"
+RESOURCE_PERMISSION_MODEL_VIEW = "PermissionModelView"
 
 # Action Constants
+ACTION_CAN_LIST = "can_list"
+ACTION_CAN_SHOW = "can_show"
```

Comment:
Might be a silly question, what's the difference between `can_read` and `can_show` ? Looks too similar, is it possible we can use `can_read` instead?

Reply:
Actually there's no `can_read` for the RoleModelView. It's something that came with FAB.  Here's the list of permissions they have for RoleModelView: 
`can show on RoleModelView, can list on RoleModelView, can download on RoleModelView, can add on RoleModelView, can delete on RoleModelView, can edit on RoleModelView, copyrole on RoleModelView`.

I got these by looking at the permissions assigned to Admin role in airflow UI. Security>List Roles

Reply:
aah .. so @jhtimmins mapped View based permission to resource based. James can you take a look a this and see if we should do the same for it ?

Reply:
this one: https://github.com/apache/airflow/commit/4cb06219dc6b2c9fb888446165605e63aa16bdaa